pkgs/development/tools/devpi-client/default.nix

```diff
@@ -1,28 +1,49 @@
-{ stdenv, pythonPackages, glibcLocales} :
+{ stdenv
+, lib
+, python
+, python3
```

Comment:
python3 is not used. Also, don't pass both `python` and `pythonPackages`. Both already contain references to each other.

Reply:
you are right, this is a leftover from the times where i've tried to get the `test_test.py` tests running but failed in providing a python version with setuptools to the tox environment